sdk/identity/azure-identity/azure/identity/_internal/aadclient_certificate.py

```diff
@@ -33,9 +33,17 @@ def __init__(
 
     @property
     def thumbprint(self) -> str:
-        """The certificate's SHA1 thumbprint as a base64url-encoded string"""
+        """The certificate's SHA1 thumbprint as a base64url-encoded string.
+
```

Comment:
return

Reply:
`@property` method, so choosing to exclude return for documentation brevity.